mixer/cmd/build.go

```diff
@@ -157,13 +184,25 @@ var buildOldFormatCmd = &cobra.Command{
 			fail(err)
 		}
 
-		fmt.Println("Backing up trimmed groups.ini for future build")
-		// Copy the new trimmed groups.ini to use for the +20 build which will
-		// not have manifest created for fully deleted bundles
-		if err = b.CopyTrimmedGroupsINI(); err != nil {
+		// TODO: Inject any extra files (certs, etc) here
+		// if err := AddBundleExtrasFiles(); err != nil {...}
+
+		ver, err := strconv.Atoi(b.MixVer)
+		if err != nil {
 			fail(err)
 		}
 
+		// Build the +20 update so we don't have to switch tooling in between
+		err = b.BuildUpdate(buildFlags.prefix, ver, buildFlags.format, buildFlags.noSigning, !buildFlags.noPublish, buildFlags.keepChroot)
+		if err != nil {
+			failf("Couldn't build update: %s", err)
+		}
+
+		// Reset upstreamversion if necessary
```

Comment:
Why do this here? It's still in the +10 build down below. Should both be doing it?

I haven't thought this through very carefully yet, but I think we probably want to pull it out of both and put it in the `mixer build upstream-format` command.